### DISCUSSION TITLE
Missed Files and Process Failed

### DISCUSSION THREAD

**Discussion by kenizl86** [pymedusa/Medusa#9592] [Q&A]
I started up my Medusa docker on my Unraid server again after letting it sit for a few months, and now that I'm adding shows I'm running into a **lot** of problems with shows I'm trying to add.

Here's a snippet of the log output for one show in particular:

`2021-05-29 05:16:20 WARNING	POSTPROCESSOR :: Missed file: /downloads/complete/[CBM] Cowboy Bebop 1-26 Complete (Dual Audio) [BDRip-720p-8bit]/[CBM]_Cowboy_Bebop_-_Session_26_-_The_Real_Folk_Blues_Part_II_[720p]_[DB52A840].mkv: Processing failed: This show isn't in your list, you need to add it before post-processing an episode`
`2021-05-29 05:16:20 WARNING	POSTPROCESSOR :: Missed file: /downloads/complete/[CBM] Cowboy Bebop 1-26 Complete (Dual Audio) [BDRip-720p-8bit]/[CBM]_Cowboy_Bebop_-_Session_25_-_The_Real_Folk_Blues_Part_I_[720p]_[815646BA].mkv: Processing failed: This show isn't in your list, you need to add it before post-processing an episode`
`2021-05-29 05:16:20 WARNING	POSTPROCESSOR :: Missed file: /downloads/complete/[CBM] Cowboy Bebop 1-26 Complete (Dual Audio) [BDRip-720p-8bit]/[CBM]_Cowboy_Bebop_-_Session_24_-_Hard_Luck_Woman_[720p]_[9C60E5AE].mkv: Processing failed: File exists. Marking it unsafe to replace. Reason: New quality is equal to current Allowed. Ignoring quality`
`2021-05-29 05:16:18 WARNING	POSTPROCESSOR :: Processing failed for /downloads/complete/[CBM] Cowboy Bebop 1-26 Complete (Dual Audio) [BDRip-720p-8bit]/[CBM]_Cowboy_Bebop_-_Session_25_-_The_Real_Folk_Blues_Part_I_[720p]_[815646BA].mkv: This show isn't in your list, you need to add it before post-processing an episode`

These are just some snippets from the log, and I skipped some other things that weren't relevant to the issue. By far the most common warning it's issuing is the `File exists. Marking it unsafe to replace. Reason: New quality is equal to current Allowed. Ignoring quality` warning. I'm not sure why it's saying this (or what it's even saying) because it hasn't downloaded a better quality, and the show folder is empty that it should be moving the files to.

Cowboy Bebop is just one example, I'm also finding the same issues with Black Lagoon and a few other shows that are downloading from torrents with the "episodes 1-12" format (where the one torrent has a full season). The files it downloaded were perfectly reasonable with no weird naming conventions at all, but it refused to process the files and wouldn't move them from my Transmission "complete" folder to my Anime folder. It's saying I "need to add it before post-processing" for a few of the episodes, but the files it's showing this error on have the exact same naming convention as all the other episodes and even came from the same torrent.

My settings haven't changed from when I stopped Medusa to when I started it again. It only seems to be having issues with older Anime. I do have the setting to wait to process until subtitles have been added, but I figured if that was a problem it would say it in the error logs.

Let me know if further information is required. Thanks!

**Comment by medariox** [comment]
Open a new issue on GitHub using the issue template and make sure to fill the debug logs. That's the only way we will be able to help you.

**Reply by kenizl86** [reply]
Okey dokes. Wasn't sure where to put the issue honestly :)
Thanks!
